msteams-platform/sbs-bots-with-sso.yml

```diff
@@ -41,7 +41,7 @@ items:
       > After downloading ngrok, sign up and install [authtoken](https://ngrok.com/download).
 
     * Visual Studio 2019 or the latest version
-    * [Microsoft 365 developer account](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) or access to Teams account with the appropriate permissions to install an app
+    * [Microsoft 365 developer account](concepts/build-and-test/prepare-your-o365-tenant.md) or access to Teams account with the appropriate permissions to install an app
```

Comment:
Should we add .md to the relative links? 

@v-habibaumme

Reply:
If we remove .md from relative link it doesn't work.